### PR TITLE
Search

### PR DESCRIPTION
**Pull request by Tunc88**
My Pull request

### CODE REVIEW

File: how-to-self/.idea/inspectionProfiles/Project_Default.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
+  <profile version="1.0">
```
Comment (joroGER) [inline how-to-self/.idea/inspectionProfiles/Project_Default.xml]: Why did you change these files?

File: how-to-self/src/components/UI/List/List.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+const list = (props) => {
+
+    const content = ["ContentObject1", "CO2","CO3","CO4","CO5", "CO6","CO7","CO8"]
+
+    return (
+        <
+            content
+        />
+    )
```
Comment (joroGER) [inline how-to-self/src/components/UI/List/List.js]: No need for so much line breaks `<content />` would do it

File: how-to-self/.idea/inspectionProfiles/Project_Default.xml (hunk 1 above)
Comment (Tunc88) [reply how-to-self/.idea/inspectionProfiles/Project_Default.xml]: i haven't change them, not minded

File: how-to-self/src/components/UI/List/List.js (hunk 2 above)
Comment (Tunc88) [reply how-to-self/src/components/UI/List/List.js]: k, sometimes i insert so many breaks that i am reminding myself when i look again into that code that i have a ToDo there

File: how-to-self/.idea/inspectionProfiles/Project_Default.xml (hunk 1 above)
Comment (joroGER) [reply how-to-self/.idea/inspectionProfiles/Project_Default.xml]: you changed them in your last commit when you created the components.

File: how-to-self/package.json [hunk 3]
```diff
@@ -42,6 +42,7 @@
     "react-router": "^4.2.0",
     "react-router-bootstrap": "^0.24.4",
     "react-router-dom": "^4.2.2",
+    "react-toggle-button": "^2.2.0",
     "redux": "^4.0.0",
```
Comment (joroGER) [inline how-to-self/package.json]: Do you actually use this dependency somewhere?

Comment (Tunc88) [reply how-to-self/package.json]: yes, in my search container xD
